tests/providers/odbc/hooks/test_odbc.py

```diff
@@ -329,6 +329,25 @@ def mock_handler(*_):
             result = hook.run("SQL", handler=mock_handler)
         assert hook_result == result
 
+    def test_query_return_serializable_result_empty(
+        self, pyodbc_row_mock, monkeypatch, pyodbc_instancecheck
+    ):
```

Comment:
```suggestion
    def test_query_return_serializable_result_empty(self, pyodbc_row_mock, monkeypatch, pyodbc_instancecheck):
```

ruff suggests us formatting the code this way